### PR TITLE
Fix typos in ISSUE_TEMPLATE

### PR DESCRIPTION
**Pull request by navarroaxel**
## Description
Please explain the changes you made here.

## Checklist

Please review this checklist before submitting a pull request.

- [ ] Code compiles correctly
- [ ] Created tests, if possible
- [x] All tests passing (`npm run test:all`)
- [x] Extended the README / documentation, if necessary

### CODE REVIEW

**Review by owenvoke — APPROVED** [review]
My poor English being changed to American English. 😭

Looks fine for consistency, but this technically isn't a typo. 👍🏻

**Comment by navarroaxel** [comment]
> Looks fine for consistency, but this technically isn't a typo. 👍🏻

@owenvoke I always forget how to spell behavior and the spell checker shown this as an error. 😁

File: .github/ISSUE_TEMPLATE.md [hunk 1]
```diff
@@ -1,12 +1,13 @@
...
 Tell us what happens instead.
 
 ## Log, debug output
 Put any log or error output here.
 
 ## Environment
-- Operating system - OSX/Linux/Windows (add version if needed)
-- NodeJS version
+- Operating system - macOS/Linux/Windows (add version if needed)
+- Node.js version (node --version)
```
Comment (bl-ue) [inline .github/ISSUE_TEMPLATE.md:12]:
```suggestion
- Node.js version (`node --version`)
```

File: .github/ISSUE_TEMPLATE.md [hunk 2]
```diff
@@ -1,12 +1,13 @@
...
+- tldr-node-client version (tldr --version)
```
Comment (bl-ue) [inline .github/ISSUE_TEMPLATE.md:13]:
```suggestion
- tldr-node-client version (`tldr --version`) 
```

**Review by bl-ue — CHANGES_REQUESTED** [review]

**Review by vladimyr — APPROVED** [review]

**Pull request merged**